src/main/java/de/tum/cit/aet/artemis/core/dto/NonLatestNonRatedResultsCleanupCountDTO.java

```diff
@@ -0,0 +1,4 @@
+package de.tum.cit.aet.artemis.core.dto;
+
+public record NonLatestNonRatedResultsCleanupCountDTO(int longFeedbackText, int textBlock, int feedback) {
```

Comment:
annotate DTOs with `@JsonInclude(JsonInclude.Include.NON_EMPTY)`

Reply:
maybe we can also stay in line with the arg names from `DataCleanupService`, i think they might be a bit more descriptive.

Reply:
added `@JsonInclude(JsonInclude.Include.NON_EMPTY)`.